calico/felix/config.py

```diff
@@ -248,6 +248,16 @@ def __init__(self, config_path):
                            "spoofing their source IP.  (For example, "
                            "unprivileged containers.)",
                            False, value_is_bool=True)
+        self.add_parameter("ClusterSize", "Total cluster host count",
+                           0, value_is_int=True)
+        self.add_parameter("ClusterType", "Cluster type",
```

Comment:
Should we remove this until we're actually filling it in?

If we do decide to it a different way then this would be redundant.